react/features/base/session/actionTypes.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * FIXME.
+ *
+ * {
+ *     type: SET_SESSION,
+ *     session: {
+ *         url: {string},
```

Comment:
We seem to be using the URL as the key in the sessions map. This has the caveat that visiting the same URL won't trigger a change. Since we already need UUIDs for CallKit, was using UUIDs as the key considered?

Reply:
Yes, UUIDs are planned as session IDs for the next step according to my discussions with @lyubomir